packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -82,11 +83,20 @@ export const StorageImage = ({
       onError,
       options: {
         accessLevel,
+        bucket,
         targetIdentityId: identityId,
         validateObjectExistence,
       },
     }),
-    [accessLevel, imgKey, identityId, onError, path, validateObjectExistence]
+    [
+      accessLevel,
+      bucket,
```

Comment:
If `bucket` is a non-memoized object will it cause any side effects in `useGetUrl`?

Reply:
I tested the object input for `bucket` on the Storage Image locally, and I don't see any different behavior in the network